build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -71,9 +71,12 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                 }
                 else
                 {
-                    // This is an old drop of latest so remove all old files to ensure a clean state
+                    Regex versionFileRegex = new Regex(@"(?<version>\d\.\d\.\d)-(?<release>.*)?");
```

Comment:
Separate channels would eliminate problems like this, but create greater complexity for install scripts. since they'd need to reason about two channels and how they map to one another